### PR TITLE
Fix v2 peek job error

### PR DESCRIPTION
**Pull request by Napoleon0621**

### CODE REVIEW

File: engine/redis_v2/engine.go [hunk 1]
```diff
@@ -218,9 +218,6 @@ func (e *Engine) Peek(namespace, queue, optionalJobID string) (job engine.Job, e
 		}
 		body = res[0].Body()
 	}
-	if err != nil {
-		return nil, err
-	}
```
Comment (git-hulk) [inline engine/redis_v2/engine.go:223]: Can add a test case to cover this fix.

Comment (Napoleon0621) [reply engine/redis_v2/engine.go:223]: ok，sure

File: engine/redis_v2/engine_test.go [hunk 2]
```diff
@@ -394,7 +394,7 @@ func TestEngine_Peek_SecondaryStorage(t *testing.T) {
 		ID:         "",
 		Body:       body,
 		TTL:        120,
-		Delay:      15,
+		Delay:      45,
```
Comment (Napoleon0621) [inline engine/redis_v2/engine_test.go:397]: It turned out to be that the job's delay in previous test case is not long enough, that it got pumped in redis before actually peeking the job. And therefore didn't reach the error code.
I fixed the test job delay to ensure that the job is still stored in DB when peeking job.

**Review by git-hulk — APPROVED** [review]

File: engine/redis_v2/engine_test.go (hunk 2 above)
Comment (git-hulk) [reply engine/redis_v2/engine_test.go:397]: ok, make sense

**Pull request merged**